templates/records/blocks/record_links_block.html

```diff
@@ -11,7 +11,9 @@
                     {% endif %}
                     <div class="record-links__item__text">
                         <p class="record-links__item__reference"><span class="sr-only">Link to record </span>FROM OUR COLLECTION: {{ item.record.reference_number }}</p>
-                        <p class="record-links__item__title"><a href="{{ item.record.url }}">{{ item.descriptive_title }}</a></p>
+                        <h2 class="record-links__item__title">
+                            <a data-component-name="{{ value.block.label }}" data-link-type="Link list" data-position="0" data-link="{{ item.descriptive_title }}" href="{{ item.record.url }}">{{ item.descriptive_title }}</a>
```

Comment:
`{{ value.block.label }}` outputs: Record Links, which is correct in terms of your note of having two different SF elements, this being the newer one. It probably is fine but maybe flag this in the ticket too.

Also, I think that the requirement of `data-position=0` maybe a mistake, I think it should be `data-position={{ forloop.counter }}` as that would should the elements position in the list, what do you think?

Reply:
👍  I've queried the count on slack with Elise but haven't heard back, also had the same query on another metric ticket. I suspect you're right and it should be counter, so the analytics picks up which cards are clicked most often (1st, 2nd or 3rd) etc but will wait to hear back from Elise

Reply:
I've just spoken to her and it needs to start at 0, so we need data-card-position="{{ forloop.counter0 }}"